bridge/README.md

```diff
@@ -6,7 +6,7 @@
 </h1>
 
 ![GitHub tag](https://img.shields.io/github/tag/svix/svix-webhooks.svg)
-[![Build Status](https://github.com/svix/svix-webhooks/workflows/Bridge%20CI/badge.svg)](https://github.com/svix/svix-webhooks/actions)
+[![Build Status](https://github.com/svix/svix-webhooks/workflows/Webhook%20Bridge%20CI/badge.svg)](https://github.com/svix/svix-webhooks/actions)
```

Comment:
This is fine, but I think I actually just missed renaming the workflow. The rest of them are all `Bridge`.

Reply:
OK, so wanna fix it properly? If so, please also remind me to update the docker page.

Reply:
Yep. I'll remind you once the other PR merges.